app/models/metric/capture.rb

```diff
@@ -234,4 +238,5 @@ def self.minutes_ago(value)
       value.to_i_with_method.seconds.ago.utc
     end
   end
+  private_class_method :minutes_ago
```

Comment:
@Fryguy should I move these into the singleton class?  I don't know which is more obvious in terms of what's private or not:

```ruby
# start of private class methods
self << class
  private

  def perf_capture_health_check(...)
  ...
end # end of private class methods
  ```